Makefile

```diff
@@ -28,3 +28,11 @@ test:
 test-coverage:
 	go test ./... -coverprofile coverage.out -covermode count
 	go tool cover -func coverage.out
+
+ng-build:
+	cd ./ui && npm install && npm run build
+
+run-binary:
+	./harbourbridge -webv2
+
+run-all:	ng-build build run-binary
```

Comment:
If this will start the UI in a single command then please update the README accordingly. If not then what's the purpose and should it be in a separate PR ?

Reply:
`run-binary` target doesn't make sense. The user should be able to run the webUI without having access to the source code and running a `make` target.

The built `harbourbridge` binary should directly be used, just like for all other commands today `data`, `schema` etc. This binary is built today via `build` and `build-static` targets. Please modify them. 

As per Shreya's suggestion, I would also recommend to put the `Makefile` relevant changes into a separate PR.

Reply:
Sure Manit. Will discuss about this with team and make it a separate PR.